tasks/configure_nginx.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: copy ssl cert
+  template: src=ssl-cert.pem dest=/etc/ssl/certs/ssl-cert.pem group=www-data mode=740
+
+- name: copy ssl key
+  template: src=ssl-cert.key dest=/etc/ssl/private/ssl-cert.key  group=www-data mode=740
```

Comment:
@bradenmacdonald Good point. But please note that there are places where templates are strictly neccessary (tarsnap is very picky about contents of a key-file).